packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -511,27 +510,49 @@ async function storePreProcessedConversionResult({
   }
 }
 
-function buildDocumentNameForPartialConversions(
-  fileName: string,
-  numChunksInPayload: number,
-  index: number
-): string {
-  if (numChunksInPayload <= 1) return fileName;
+function buildDocumentNameForPartialConversions(fileName: string, index: number): string {
+  const paddedIndex = index.toString().padStart(2, "0");
```

Comment:
Can't we have more than 99 files (thinking we can change the chunk size in the future)? Maybe set it for 3 digits just in case, so the nomenclature doesn't need changing in such scenario?

Reply:
ok